puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-infra-jenkins-jobs-update.yml

```diff
@@ -0,0 +1,10 @@
+- job:
+    name: foreman-infra-jenkins-jobs-update
+    project-type: 'pipeline'
+    sandbox: true
+    scm:
```

Comment:
Is this needed if cloning inside the pipeline?

Reply:
Probably not, went ahead and removed.